src/character.cpp

```diff
@@ -2001,3 +2001,36 @@ bool Character::pour_into( vehicle &veh, item &liquid )
     }
     return true;
 }
+
+long Character::ammo_count_for( const item &gun )
+{
+    // INT_MAX instead of LONG_MAX because some code may not expect > INT_MAX
```

Comment:
I believe it has to be a [named constant](https://github.com/CleverRaven/Cataclysm-DDA/pull/16901/files#diff-d13ed72411aea98b2ed21e2cb17387ebR572) everywhere.